Code/GraphMol/RGroupDecomposition/RGroupDecompData.h

```diff
@@ -326,6 +326,15 @@ struct RGroupDecompData {
     }
 
     addAtoms(core, atomsToAdd);
+    for (const auto &rlabels : atoms) {
+      auto atom = rlabels.second;
+      if (atom->hasProp(RLABEL)) {
+        atom->clearProp(RLABEL);
+      }
+      if (atom->hasProp(RLABEL_TYPE)) {
+        atom->clearProp(RLABEL_TYPE);
+      }
```

Comment:
It's safe to call `clearProp()` on a property that doesn't exist, so this can be:
```suggestion
      atom->clearProp(RLABEL);
      atom->clearProp(RLABEL_TYPE);
```

Reply:
Ah, good to know; done.